### DISCUSSION TITLE
Be able to open user profile through recent messages tab

### DISCUSSION THREAD

**Discussion by Dannyaa** [ppy/osu#16191] [Ideas]
You can open someones profile by clicking their name in the dm's, but not through the tab itself. This would be a convenient addition for when there's no dm's to open their profile with. 
![S3LaxaG](https://user-images.githubusercontent.com/57819431/146875905-4823ca6f-c1ec-48bb-8896-ad42f91e754c.png)

**Comment by peppy** [comment]
If anything, this would need to be limited to a second click when the tab is already selected. Also it should only work on the avatar. There's no world in which that text there should have a hover/link effect.
